docs/learn/smart-contract-developers/create-nft-collection-with-lsp7-tokenId.md

```diff
@@ -0,0 +1,956 @@
+- the total supply of the token
+- the [Token Type](../../standards/tokens/LSP4-Digital-Asset-Metadata.md#types-of-digital-assets)
+- the [LSP4 Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md#lsp4metadata)
+
+This is just an example but you may want to set a deployment of other parameters such as the [LSP4 Creators array](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md#lsp4creators).
+
+### Imports
+
+Let's start creating an `LSP7Token.sol` file in the `contracts` folder and importing:
+
+- the [`LSP7DigitalAsset.sol`](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP7DigitalAsset/LSP7DigitalAsset.sol) contract
+- the [`LSP4DigitalAssetMetadata.sol`](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP4DigitalAssetMetadata/LSP4DigitalAssetMetadata.sol) contract
+- the [`_LSP4_SUPPORTED_STANDARDS_KEY`](../../standards//tokens/LSP4-Digital-Asset-Metadata.md#supportedstandardslsp4digitalasset) constant
+- the [`_LSP4_SUPPORTED_STANDARDS_VALUE`](../../standards//tokens/LSP4-Digital-Asset-Metadata.md#supportedstandardslsp4digitalasset) constant
+- the [`_LSP4_METADATA_KEY`](../../standards//tokens/LSP4-Digital-Asset-Metadata.md#lsp4metadata) constant
+- the [`_LSP8_REFERENCE_CONTRACT`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#lsp8referencecontract) constant
+
+```typescript title="contracts/LSP7Token.sol"
+import { LSP7DigitalAsset } from '@lukso/lsp-smart-contracts/contracts/LSP7DigitalAsset/LSP7DigitalAsset.sol';
+import { LSP4DigitalAssetMetadata } from '@lukso/lsp-smart-contracts/contracts/LSP4DigitalAssetMetadata/LSP4DigitalAssetMetadata.sol';
+
+import {
+  _LSP4_SUPPORTED_STANDARDS_KEY,
+  _LSP4_SUPPORTED_STANDARDS_VALUE,
+  _LSP4_METADATA_KEY
+} from '@lukso/lsp-smart-contracts/contracts/LSP4DigitalAssetMetadata/LSP4Constants.sol';
+import { _LSP8_REFERENCE_CONTRACT } from '@lukso/lsp-smart-contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8Constants.sol';
+<!-- prettier-ignore-end -->
+```
+
+### Contract
+
+At deployment, we will need to set the following parameters:
+
+- `name_`: the name of the token
+- `symbol_`: the symbol of the token
+- `owner_`: the address that will be able to update the contract metadata (except the symbol, name and the LSP4TokenType)
+- `receiverOfInitialTokens_`: the address that will receive the minted tokens at deployment
+- `[lsp4TokenType\_](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md#lsp4tokentype): the token type of the token
+- [`isNonDivisible_`](../../contracts/contracts/LSP7DigitalAsset/LSP7DigitalAsset.md#decimals): whether the token is divisible or not
+- `totalSupply_`: the total supply of the token
+- `lsp4MetadataURI_`: the [LSP4 Metadata](../../standards//tokens/LSP4-Digital-Asset-Metadata.md#lsp4metadata)
+
+```typescript title="contracts/LSP8Token.sol"
```

Comment:
```suggestion
```typescript title="contracts/LSP7Token.sol"
```

Reply:
Maybe we should rename these `LSP7SubCollection` to improve the understanding of the reader?